analyses/snv-callers/scripts/02-merge_callers.R

```diff
@@ -67,6 +67,69 @@ option_list <- list(
 # Parse options
 opt <- parse_args(OptionParser(option_list = option_list))
 
+vaf_filter <- opt$vaf_filter # get out of opt list for sql
+
+############################## Custom Functions ################################
+
+#' Split multinucleotide variants into single nucleotide calls
+#'
+#' @param mnv_tbl a table containing MNVs (may be from an sql connection)
+#'
+#' @return a data frame of SNVs derived from the MNVs
+#'   (does not contain any of the original SNVs)
+split_mnv <- function(mnv_tbl) {
+  mnv_df <- mnv_tbl %>%
+    dplyr::filter(Variant_Type %in% c("DNP", "TNP", "ONP")) %>%
+    as.data.frame() %>%
+    # temp_id is not currently used, but will be handy for reconstituting MNVs
+    dplyr::mutate(temp_id = dplyr::row_number()) %>%
+    # lancet adds a base to the start of MNV Allele Calls
+    dplyr::mutate(
+      Allele = dplyr::case_when(
+        stringr::str_length(Allele) == stringr::str_length(Reference_Allele) ~
```

Comment:
Can you add a comment or two here to explain the logic of this `case_when` a bit more?

Reply:
I did... its a hack because lancet is strange.